insights/parsers/corosync.py

```diff
@@ -1,47 +1,171 @@
 """
-CoroSyncConfig - file ``/etc/sysconfig/corosync``
-=================================================
-
-This parser reads the ``/etc/sysconfig/corosync`` file.  It uses the
-``SysconfigOptions`` parser class to convert the file into a dictionary of
-options.  It also provides the ``options`` property as a helper to retrieve
-the ``COROSYNC_OPTIONS`` variable.
-
-Sample data::
-
-    # Corosync init script configuration file
-
-    # COROSYNC_INIT_TIMEOUT specifies number of seconds to wait for corosync
-    # initialization (default is one minute).
-    COROSYNC_INIT_TIMEOUT=60
+Parsers for the Corosync Cluster Engine configurations
+======================================================
 
-    # COROSYNC_OPTIONS specifies options passed to corosync command
-    # (default is no options).
-    # See "man corosync" for detailed descriptions of the options.
-    COROSYNC_OPTIONS=""
 
-Examples:
+Parsers included in this module are:
 
-    >>> csconfig = shared[CoroSyncConfig]
-    >>> 'COROSYNC_OPTIONS' in csconfig.data
-    True
-    >>> csconfig.options
-    ''
+CoroSyncConfig - file ``/etc/sysconfig/corosync``
+-------------------------------------------------
 
+CorosyncConf - file ``/etc/corosync/corosync.conf``
```

Comment:
To distinguish the new Parser `CorosyncConf` with the original one, I think better`deprecate` the old one `CoroSyncConfig` and move it to the [sysconfig.py](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/sysconfig.py) and rename it as `CorosyncSysconfig`. 

For all `/etc/sysconfig/xxx` configuration files, I prefer to put them into [sysconfig.py](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/sysconfig.py)

@bfahr , @psachin - Any thoughts?